tests/acceptance/staff-test.js

```diff
@@ -522,7 +522,12 @@ describe('Acceptance: Staff', function () {
                 // Save changes
                 await click('[data-test-save-button]');
 
-                expect(find('[data-test-save-button]').textContent.trim(), 'save button text').to.equal('Saved');
+                // Since we reset save status so there's no on-screen indication
+                // that we've had a save, check the request was fired instead
+                let [lastRequest] = this.server.pretender.handledRequests.slice(-1);
+                let params = JSON.parse(lastRequest.requestBody);
+
+                expect(params.users[0].name).to.equal('Test User');
```

Comment:
@kevinansfield Not 💯 here if this is the best way to test the saved status since we turned on the auto-reset for all buttons, so checking button text as "Saved" is not gonna work here anymore. Copied this pattern from just below where we already checked the save status with request, lmk tho if this is not ideal and should be changed.

Reply:
👍 LGTM